service/src/main/kotlin/app/cash/backfila/dashboard/CloneBackfillAction.kt

```diff
@@ -90,8 +90,7 @@ class CloneBackfillAction @Inject constructor(
   @Post("/backfills/{id}/clone")
   @RequestContentType(MediaTypes.APPLICATION_JSON)
   @ResponseContentType(MediaTypes.APPLICATION_JSON)
-  // TODO allow any user
-  @Authenticated(capabilities = ["users"])
+  @Authenticated(allowAnyUser = true)
```

Comment:
This seems wrong? is `@Authenticated` the same as `@Authenticated(capabilities = ["users"])`?

Reply:
The TODO comment seemed to imply that the desired eventual configuration was any authenticated user but since that didn't exist in Misk until now, allowing the entire `users` group was what was done. That TODO is on every similarly configured annotation in the repo. Open to your / @shellderp 's call on what you want access configured as but this is what I understood from the TODO comment.